webpack.static-viz.config.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
 
   entry: {
     "lib-static-viz": {
-      import: "./static-viz/index.js",
+      import: "./static-viz/index.ts",
```

Comment:
Not sure what's with this change. I don't see you renaming this file to `.ts` in this PR.

Reply:
It's not intended, fixed